app.js

```diff
@@ -0,0 +1,10 @@
+document.querySelectorAll('.menu-btn').forEach((item) => {
+  item.addEventListener('click', () => {
+    const menu = document.getElementById('menu-mobile');
+    if (menu.classList.contains('active')) {
+      menu.classList.remove('active');
+    } else {
+      menu.classList.add('active');
+    }
+  });
```

Comment:
- [ ] Great job with your design and functionalities 👏 . As stated in the project requirements, kindly make sure when a user clicks any of the mobile menu options, the correct part of the page is displayed. 🙏 
- [ ] As stated in the project requirements, kindly make sure when a user clicks any of the mobile menu options, the mobile menu disappears. 🙏